Dockerfile

```diff
@@ -13,6 +13,7 @@ WORKDIR /opt/${REPONAME}
 COPY . .
 
 RUN cp /opt/epics/RELEASE configure/RELEASE
+RUN rm -rf .git/
```

Comment:
This adds another caching layer, which shouldn't be needed. It may slow down the build a tiny bit.

Is there any special reason you made this another `RUN` layer?

Reply:
Mostly because the `static-build` stage has separate RUNs for commands which weren't as closely related.
I'd be okay with moving this into the RUN before it.